recipes/_configure_limits.rb

```diff
@@ -0,0 +1,11 @@
+# coding: UTF-8
+#
+# Cookbook Name:: cerner_splunk
+# Recipe:: _configure_limits
+#
+# Configures the limits.conf settings for the system
+
+splunk_template 'system/limits.conf' do
+  stanzas node['splunk']['config']['limits']
```

Comment:
Added a default to nil (it actually wasn't required to be run previously, although the limits file does always get created). 

95bc704

I could add a check to ensure the file isn't created (or is removed) when there isn't content to populate it. It actually seems a bit safer to me to always create the file (potentially to have it be empty when attributes are removed). The removal portion I think is the tricky part, normal templates don't really behave that way, although they do have a delete action.

I think being under config makes sense, I might not be following though with the comparison to apps and monitors. Portions of the limits configuration apply to many of the different components in splunk.